global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "2.1.500"
+    "version": "2.1.818"
```

Comment:
you could potentially remove "sdk"... so we can open it with slightly different versions

Reply:
Will do - we were talking about interning this - so the SDK stuff might well end up being moot anyway 👍